index.html

```diff
@@ -52,8 +52,8 @@
       </div>
     </div>
     <script src="https://cdn.jsdelivr.net/gh/cryptonomic/conseiljs/dist-web/conseiljs.min.js"
-        integrity="sha384-IDwJO+ahQcXoDx++lRH6FPOk8JDHv9RbB2HBME19y52wQ5pi+HtT/ndp8dpdx83f"
-        crossorigin="anonymous" type="text/javascript"></script>
+        integrity="sha384-9KFoSwB7AGW9/uAX7v//0xQaYLNB6ZV9+MXhm5WJAEFiyye4XGdtCd2WNXqU5x3B"
```

Comment:
Fair point, this indeed must be updated every time there is a new release of the minified js file.